pycbc/results/scatter_histograms.py

```diff
@@ -385,16 +395,91 @@ def create_marginalized_hist(ax, values, label, percentiles=None,
             ax.set_xlim(xmin, xmax)
 
 
+def set_marginal_histogram_title(ax, fmt, color, label=None, rotated=False):
+    """ Sets the title of the marginal histograms.
+
+    Parameters
+    ----------
+    ax : Axes
+        The `Axes` instance for the plot.
+    fmt : str
+        The string to add to the title.
+    color : str
+        The color of the text to add to the title.
+    label : str
+        If title does not exist, then include label at beginning of the string.
+    rotated : bool
+        If `True` then rotate the text 270 degrees for sideways title.
+    """
+
+    # get rotation angle of the title
+    rotation = 270 if rotated else 0
+
+    # get how much to displace title on axes
+    xscale = 1.05 if rotated else 0.0
+    if rotated:
+        yscale = 1.0
+    elif len(ax.get_figure().axes) > 1:
+        yscale = 1.15
+    else:
+        yscale = 1.05
+
+    # get class that packs text boxes vertical or horizonitally
+    packer_class = offsetbox.VPacker if rotated else offsetbox.HPacker
+
+    # if no title exists
+    if not hasattr(ax, "title_boxes"):
+
+        # create a text box
+        title = "{} = {}".format(label, fmt)
```

Comment:
This is slightly off topic, but since you're touching this code... I've been wanting to change the title string for awhile now. Because we have `{} = {}`, the labels often bleed over into the left plot in the corner plots. Can you change this to be `{}\\{}`, so that the label is on a line above, with the values a line below? (It might be `\n` instead of `\\`; can't remember.) That should help the problem quite a bit. For the multiple comparisons, just put each value on a new line.

Reply:
I already fixed it so they're all right-aligned instead of centered so it doesn't go over into the plots. In fact the plot will expand too if its too long instead of cutting it off. Look at the example and the distance column.

Reply:
Oh, I missed that. Ok, left justifying works too.